plugins/powertable/plugin.py

```
@@ -0,0 +1,46 @@
+    description = "PowerTable plugin"
+    is_core = False
+
+    @staticmethod
+    def migrations_path() -> Path:
+        return Path(__file__).parent / "migrations"
+
+    def on_load(self, ctx: dict[str, Any]) -> None:
+        services = ctx.get("services")
+        services.register("powertable.echo", lambda x: x)
+
+    def register_routes(self, app: FastAPI, router: APIRouter) -> None:
```

Ruff (0.12.2)

Unused method argument: `app`

(ARG002)